security/kubernetes-default-deny.md

```diff
@@ -58,6 +62,57 @@ spec:
   - Egress
 ```
 
+The above policy applies to all pods, hosts and endpoints, including Kubernetes and {{site.prodname}} control plane pods. Such policy has the potential to interrupt your cluster if you have not the correct "Allow" policies in place to keep the control plane functioning properly.
+
+As a best practice we recommend to use the following examples depending on your {{site.prodname}} installation method.
+
+{% tabs %}
+<label:Manifest,active:true>
+<%
+```yaml
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: deny-app-policy
+spec:
+  selector: 'projectcalico.org/namespace != "kube-system"'
+  types:
+  - Ingress
+  - Egress
+  egress:
+  # allow all namespaces to communicate to DNS pods
+  - action: Allow
+    protocol: UDP
+    destination:
+      selector: 'k8s-app == "kube-dns"'
+      ports:
+      - 53
+```
+%>
+<label:Operator>
+<%
+```yaml
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: deny-app-policy
+spec:
+  selector: 'projectcalico.org/namespace != "kube-system" || projectcalico.org/namespace != "calico-system"'
```

Comment:
I think this should this be an AND rather than an OR?

Reply:
Oops! :)